### PR TITLE
Db setup

### PR DESCRIPTION
**Pull request by jbratton**
I wasn't sure about how best to implement an Organization's owner with Active Record magic, but what I tried seems to work.

Also not sure about GlobalList < RecipientList and the single table inheritance magic.

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (bentona) [inline config/environments/development.rb:18]: You should add an entry for our heroku URL in `production.rb` also

**Comment by bentona** [comment]
What is GlobalList for?

**Comment by jbratton** [comment]
`GlobalList` is the "All Recipients" list for an `Organization`. Maybe it should be `AllRecipientsList`. That whole part is still a little confusing to me. I'll have to mess with it in the rails console or something and then probably fix it.

**Comment by bentona** [comment]
Is it different than just `organization.recipients.all`?

**Comment by jbratton** [comment]
It is not different. I was following @chamblin's advice as discussed in issue #2 which would allow the "All Recipients" list to be handled as an instance of a `RecipientList` subclass instead of needing special cases. Or that was my understanding of the reasoning behind using single table inheritance there anyway.

**Comment by jbratton** [comment]
I'm going to break this down into smaller changes/PRs.

**Pull request closed without merging**